crates/storage/provider/src/providers/blockchain_provider.rs

```diff
@@ -279,7 +279,7 @@ impl<N: ProviderNodeTypes> BlockchainProvider2<N> {
         // the in-memory state.
         let last_block_body_index = provider
             .block_body_indices(last_database_block_number)?
-            .ok_or_else(|| ProviderError::BlockBodyIndicesNotFound(last_database_block_number))?;
+            .ok_or(ProviderError::BlockBodyIndicesNotFound(last_database_block_number))?;
```

Comment:
this should not be changed?
```suggestion
            .ok_or_else(|| ProviderError::BlockBodyIndicesNotFound(last_database_block_number))?;
```

Reply:
yeah idk, 
cargo warning

```
warning: unnecessary closure used to substitute value for `Option::None`
   --> crates/storage/provider/src/providers/blockchain_provider.rs:280:37
    |
280 |           let last_block_body_index = provider
    |  _____________________________________^
281 | |             .block_body_indices(last_database_block_number)?
282 | |             .ok_or_else(|| ProviderError::BlockBodyIndicesNotFound(last_database_block_number))?;
    | |_______________________________________________________________________________________________^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_lazy_evaluations
    = note: `#[warn(clippy::unnecessary_lazy_evaluations)]` on by default
help: use `ok_or` instead
    |
282 |             .ok_or(ProviderError::BlockBodyIndicesNotFound(last_database_block_number))?;
```